app/code/Magento/Customer/Controller/Account/CreatePassword.php

```diff
@@ -83,6 +91,15 @@ public function execute()
 
             $this->confirmByToken->execute($resetPasswordToken);
 
+            try {
+                // Extend token validity to avoid expiration while this form is
+                // being completed by the user.
+                $customer = $this->getByToken->execute($resetPasswordToken);
+                $this->accountManagement->changeResetPasswordLinkToken($customer, $resetPasswordToken);
+            } catch (\Exception $exception) {
+                // Intentionally ignoring failures here
```

Comment:
1. Catching all types exceptions isn't really good thing. I believe we shouldn't catch PDOException here, for instance? Can we catch there only exceptions that we really expect?

2. Empty catch block also not a good thing, usually it just masks the problems. Which failures we expect here? Why should we ignore them? Should we add some message or maybe log them in some case?